[NrpGhpKJ] FAQ

AlexeyFrunza opened the pull request:
-- добавлен widget FaqWidget
-- для статичных данных в папке widgets/faq/ создана папка consts
-- виджет адаптирован под мобильные и десктопные устройства

Commit:
[NrpGhpKJ] FAQ
-- Добавлена секция faq

Commit:
[NrpGhpKJ] FAQ
-- initial state

Commit:
[NrpGhpKJ] FAQ
-- добавлен widget FaqWidget
-- для статичных данных в папке widgets/faq/ создана папка consts
-- виджет адаптирован под мобильные и десктопные устройства

weekkkk commented on public/images/faq-grid.webp:
зачем

weekkkk commented on src/app/assets/css/tailwind.css:
```diff
@@ -6,11 +6,33 @@
 @layer components {
 }
+
+@layer utilities {
+  .i-custom-arrow-down {
+    background-image: url("../icons/arrow-down.svg");
+    width: 3rem;
+    height: 3rem;
+    display: inline-block;
+    background-size: cover;
+  }
+  .i-custom-arrow-up {
+    background-image: url("../icons/arrow-up.svg");
+    width: 3rem;
+    height: 3rem;
+    display: inline-block;
+    background-size: cover;
+  }
+}
```
зачем

weekkkk replied:
https://github.com/nuxt/icon?tab=readme-ov-file#custom-local-collections почитай

Commit:
[NrpGhpKJ] FAQ
-- в файле tailwind.css удален @layer utilities
-- вместо этого используется класс 'xi-i-...'

AlexeyFrunza committed:
Merge branch 'develop' into feature/NrpGhpKJ

AlexeyFrunza committed:
Merge branch 'develop' into feature/NrpGhpKJ

AlexeyFrunza commented:
поправил конфликты

weekkkk commented on public/images/faq-grid.webp:
зачем

Commit:
[NrpGhpKJ] FAQ
-- удален файл faq-grid.webp
-- вместо него используется faq-grid.svg

AlexeyFrunza commented:
поправил
